google/cloud/compute/accelerator_types/v1/internal/accelerator_types_option_defaults.cc

```diff
@@ -50,8 +50,9 @@ Options AcceleratorTypesDefaultOptions(Options options) {
                        AcceleratorTypesBackoffPolicyOption>()) {
     options
         .set<compute_accelerator_types_v1::AcceleratorTypesBackoffPolicyOption>(
-            ExponentialBackoffPolicy(std::chrono::seconds(1),
-                                     std::chrono::minutes(5), kBackoffScaling)
+            ExponentialBackoffPolicy(
```

Comment:
Unrelated to this SHA update PR...

This is strange. I would have expected `generate-libraries` to fail at HEAD.